wasmRunner.spec.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import { compileWat } from "./compile-wat.js";
+// @ts-ignore
```

Comment:
You can also change this to `.test.ts` and use `ts-jest` (or `babel-jest` with typescript).

This works tho!